client/decrypt/decrypt.go

```diff
@@ -0,0 +1,98 @@
+package decrypt
+
+import (
+	"encoding/hex"
+	"fmt"
+	"github.com/btcsuite/btcd/btcec"
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/crypto/keyring"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	"github.com/tendermint/tendermint/libs/cli"
+	"io/ioutil"
+)
+
+const (
+	FlagCipherText     = "text"
```

Comment:
Why don't we use "cipher-text"? Although there is no flag under "text", the word "text" is used in output flag. But it's not that important

Reply:
That's good idea.
I fixed it.
Thank you :)